test_scripts/Polices/build_options/016_ATF_P_TC_Timeout_Countdown_Start_PTU_Response_HTTP.lua

```diff
@@ -120,11 +114,11 @@ function Test:TestStep_Sending_PTS_to_mobile_application()
     return true
   end
 
-  EXPECT_NOTIFICATION("OnSystemRequest", { requestType = "HTTP", fileType = "JSON"})
+  EXPECT_NOTIFICATION("OnSystemRequest", { requestType = "HTTP", fileType = "BINARY"})
```

Comment:
@dboltovskyi, Timeout is added, but prefer to leave TestCases:DelayedExp(time_wait) in case of SDL problem, for example sends OnSystemRequest more that once during time_wait.

Please check 7593939284900211ec8672a429d27571a99df70f